prefect_census/client.py

```diff
@@ -0,0 +1,93 @@
+"""Module containing client for interacting with the Census API"""
+from httpx import AsyncClient, Response
+
+
+class CensusClient:
+    """
+    Client for interacting with the Census API.
+
+    Args:
+        api_key: API key to authenticate with the Census API.
+    """
+
+    def __init__(self, api_key: str):
+        self._closed = False
+        self._started = False
+
+        self.client = AsyncClient(
+            base_url="https://app.getcensus.com",
```

Comment:
Since the `/api/v1` is common across API routes, it might make sense to include it in the `base_url`.
```suggestion
            base_url="https://app.getcensus.com/api/v1",
```